.github/workflows/nightly-e2e.yaml

```
@@ -75,7 +75,9 @@ jobs:
           if-no-files-found: ignore
 
   cloud-experimental-e2e:
-    if: github.repository == 'NVIDIA/NemoClaw'
+    # DISABLED: Landlock /sandbox writability regression + CLI/docs command
+    # reference drift. Re-enable once both are fixed.
+    if: false # was: github.repository == 'NVIDIA/NemoClaw'
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)